setup.cfg

```diff
@@ -105,30 +101,27 @@ universal = 1
 pins =
     ###
     # mutually compatible pyviz-maintained packages
-    bokeh = 1.0.0dev6
-    holoviews = 1.11.0a8
-    hvplot = 0.2.1
-    geoviews = 1.5.4a13
-    datashader = 0.6.8
-    panel = 0.1.2
-    param = 1.8.1
-    paramnb = 2.0.4
-    parambokeh = 0.2.3
+    bokeh = 1.0.4
+    holoviews = 1.11.2
+    hvplot = 0.4.0
+    geoviews = 1.6.2
+    datashader = 0.6.9
+    panel = 0.4.0
+    param = 1.8.2
     ###
     # avoid tab-completion issue in ipython=6
     ipython = 5.*
     # for datashader compatibility
-    dask = 0.18.2
+    dask = 1.1.0
```

Comment:
Is this available on defaults? If not, you'll either need to use e.g. conda-forge (in CI and in instructions for users), or copy dask onto the pyviz channel...

Reply:
(By defaults I mean the anaconda main channel, sorry.)

Reply:
I'll probably relax the pin instead.

Reply:
>If not, you'll either need to use e.g. conda-forge (in CI and in instructions for users), or copy dask onto the pyviz channel...

Actually we also want latest fastparquet, I'll add conda-forge to the CI now and then hope that the conda folks update fastparquet and dask along everything else this week.

Reply:
Please don't depend on c-f unless its needed.

Reply:
Are we depending on things that are in the most recent versions?

Reply:
It is until fastparquet and dask are updated in defaults unless we want to force user to use old pandas.

Reply:
Altenatively we could also stop pinning pyviz as strictly as we have been.

Reply:
I would vote for not pinning as tightly. Or pinning between a few versions

Reply:
I'll leave the list of strict pins in the setup.cfg for now, so we can potentially a heavily pinned ``pyviz-tutorial`` package for running live tutorials.